src/reporter/query_1T1ENA.py

```diff
@@ -97,6 +98,7 @@ def query_1T1ENA(entity_id,   # In Path
 
 
 def query_1T1ENA_value(*args, **kwargs):
+    pdb.set_trace()
```

Comment:
@daminichopra slight oversight here? it looks like you removed `import pdb` in an earlier commit, so we should get rid of this too otherwise QL will die of a horrible death while running the tests---gosh you gotta love dynamic languages...